model/feature_flags.go

```diff
@@ -59,7 +59,7 @@ func (f *FeatureFlags) SetDefaults() {
 	f.CollapsedThreads = true
 	f.EnableRemoteClusterService = false
 	f.AppsEnabled = false
-	f.PluginIncidentManagement = "1.16.1"
+	f.PluginIncidentManagement = "1.20.0"
```

Comment:
Wrong one. Needs to be `PluginPlaybooks`. You can delete this line and the old feature flag if we don't want to control the old versions with split.io.